ideas/87-new-protocol.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: 87-new-protocol
 title: New Status App communication protocol
-status: Limbo
+status: In Progress
```

Comment:
Can you mark this as 'Active'? That's the state defined in https://github.com/status-im/ideas/blob/master/_data/statuses.yaml due to some Jekyll stuff